src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -375,11 +365,21 @@ export const VenueLandingPage: React.FunctionComponent<VenueLandingPageProps> =
           eventPaidSuccessfully={eventPaidSuccessfully}
         />
       )}
-      <AuthenticationModal
-        show={isAuthenticationModalOpen}
-        onHide={closeAuthenticationModal}
-        afterUserIsLoggedIn={() => setShouldOpenPaymentModal(true)}
-        showAuth={AuthOptions.register}
+      {!isLoggedIn && (
+        <AuthenticationModal
+          show={isAuthenticationModalOpen}
+          onHide={closeAuthenticationModal}
+          headerMessage={"You need an account to join this party"}
+          afterUserIsLoggedIn={() => setShouldOpenPaymentModal(true)}
+          showAuth={AuthOptions.initial}
+        />
+      )}
```

Comment:
I don't believe we ever want to conditionally render/unrender modals. I believe the intent is that the `show` prop is used for this purpose. So probably 
```suggestion
      <AuthenticationModal
        show={!isLoggedIn && isAuthenticationModalOpen}
        onHide={closeAuthenticationModal}
        headerMessage={"You need an account to join this party"}
        afterUserIsLoggedIn={() => setShouldOpenPaymentModal(true)}
        showAuth={AuthOptions.initial}
        />
```